src/utils/billing/index.ts

```diff
@@ -79,10 +81,45 @@ export function isSubscriptionCancelled(subscription?: Subscription) {
     return subscription && subscription.attributes.isCancelled;
 }
 
+// isPartOfFamily return true if the current user is part of some family plan
+export function isPartOfFamily(familyData?: FamilyData): boolean {
+    familyData = familyData ?? getFamilyData();
+    return familyData && familyData.members && familyData.members.length > 0;
```

Comment:
this is returning `undefined` when `familyData` is `undefined`

this will return `undefined`  and not `false` when `familyData` is `undefined` 

we can simple wrap the whole  thing in a Boolean()   to fix this 
```
return Boolean(familyData 
          && familyData.members 
          && familyData.members.length > 0);
```